lib/modules/manager/nuget/util.ts

```diff
@@ -52,13 +52,19 @@ export async function getConfiguredRegistries(
   }
 
   const packageSources = nuGetConfig.childNamed('packageSources');
+
   if (!packageSources) {
+    // If there are no packageSources, don't even look for any
+    // disabledPackageSources
+    // Even if NuGet default source (nuget.org) was among the
+    // disabledPackageSources, Renovate will default to the default source
```

Comment:
should we issue a warning if default source is disabled?

Reply:
@viceice Do you mean if default source is disabled and there are no packageSources, or if default source is disabled and there are other packageSources?

Reply:
@viceice It is a perfectly valid case to disable the default source. For instance, in a corporate environment, you might not want to query directly nuget.org, but a proxy for security reasons.

I don't think we should issue a warning. Feel free to reopen if you don't agree as I'm not sure if your comment was really a question or more a suggestion (in that case, I'll implement the warning).